### ISSUE TITLE
W504 false positive for type annotation `...`

### ISSUE THREAD

**Issue by asottile** [PyCQA/pycodestyle#874]
```python
def f(
    match: str = ...
):
    ...
```

```console
$ pycodestyle t.py --select=W504
t.py:2:18: W504 line break after binary operator
$ pycodestyle --version
2.5.0
```

discovered in https://github.com/pytest-dev/pytest/pull/5593/files
